TCAT/Controllers/AllStopsTableViewController.swift

```diff
@@ -152,13 +152,17 @@ class AllStopsTableViewController: UIViewController {
                             userDefaults.set(encodedObject, forKey: Constants.UserDefaults.allBusStops)
                         } catch let error {
                             self.printClass(context: "\(#function) error", message: error.localizedDescription)
+                            let payload = GetErrorPayload(location: "\(self) Get All Stops", type: "\((error as NSError).domain)", description: error.localizedDescription, url: nil)
```

Comment:
can we separate these across multiple lines like so:
```
let payload = GetErrorPayload(
  location: ,
  type: ,
  ...
)
```

Reply:
Let's do this for all